packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -70,6 +84,11 @@ const {
   selectedOption: computed(() => props.modelValue),
   activeOption,
   onSelect: (selectedOption) => {
+    if (selectedOption === CHECK_ALL_ID) {
+      checkAll.value?.change(!checkAll.value.state.modelValue);
```

Comment:
I don't see where in checkAll.ts you invert the logic of the change event?

Reply:
Its the change with `satisfies Partial<OnyxCheckboxProps>` :) With this, we don't need the `!` here because TypeScript will know that the modelValue is not undefined

Reply:
the ! is meant to invert the boolean: true to false and false to true

Reply:
Ooh I am so lost :D Sorry!